commons-math-legacy/src/main/java/org/apache/commons/math4/legacy/analysis/interpolation/BicubicInterpolatingFunction.java

```diff
@@ -181,6 +226,75 @@ public boolean isValidPoint(double x, double y) {
             y > yval[yval.length - 1]);
     }
 
+    /**
+     * @return the first partial derivative respect to x.
+     * @throws NullPointerException if the internal data were not initialized
+     * (cf. {@link #BicubicSplineInterpolatingFunction(double[],double[],double[][],
```

Comment:
`BicubicSplineInterpolatingFunction` does not exist. Please update all 5 references in the javadoc of this class to `BicubicInterpolatingFunction`

Reply:
Done!